sdk/core/azure-core/tests/test_pipeline.py

```diff
@@ -204,6 +204,18 @@ def test_format_url_double_query():
     assert formatted == "https://bing.com/path/subpath?query=testvalue&x=2ndvalue&a=X&c=Y"
 
 
+def test_format_url_braces_with_dot():
```

Comment:
Did this test repro the "infinite loop" issue?

Reply:
Yes. We reproed the infinite loop.